### PR TITLE
[SolaxX1] Updated safety descriptions

### PR DESCRIPTION
**Pull request by SteWers**
## Description:

Updated safety descriptions

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] The code change is tested and works with Tasmota core ESP8266 V.2.7.4.9
  - [ ] The code change is tested and works with Tasmota core ESP32 V.2.0.6
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

### CODE REVIEW

**Comment by Jason2866** [comment]
Noob question, for what is this used/good for in general?

**Comment by SteWers** [comment]
In general the driver is used to read out data form Solax solar inverters via RS485 interface. The inverters from Solax are sold all over the world. Every country has individual regulations for connecting them to the mains. So Solax put configurations to match that regulations. Which standard is used by the inverter can be read out as an integer value. The list contains the names of the standards according to the integer value. More details can be found in the [docs](https://tasmota.github.io/docs/SolaX-X1/).

**Comment by arendst** [comment]
Ah you're writing a book....

File: tasmota/tasmota_xnrg_energy/xnrg_12_solaxX1.ino [hunk 1]
```diff
@@ -48,9 +48,10 @@ const char kSolaxError[] PROGMEM =
 
 #ifdef SOLAXX1_READCONFIG
 const char kSolaxSafetyType[] PROGMEM =
-  "VDE0126|ARN4105|AS4777_AU|G98/1|C10/11|OVE/ONORME8001|EN50438_NL|EN50438_DK|CEB|CEI021|NRS097_2_1|"
-  "VDE0126_Gr_Is|UTE_C15_712|IEC61727|G99/1|VDE0126_Gr_Co|France_VFR2014|C15_712_is_50|C15_712_is_60|"
-  "AS4777_NZ|RD1699|Chile|EN50438_Ireland|Philippines|Czech_PPDS|Czech_50438";
+  "VDE 0126|VDE-AR-N 4105|AS 4777|G98|C10/11|ÖVE/ÖNORM E 8001|EN 50438 NL|EN 50438 DK|CEB|CEI021|NRS 097-2-1|VDE 0126 Greece/Iceland|"
```
Comment (arendst) [inline tasmota/tasmota_xnrg_energy/xnrg_12_solaxX1.ino:51]: pls remove all this and if you feel you need to report something just display the number and let users search the solaxx documentation for it's meaning. Saves 1k code space to be used for important functions.

Comment (SteWers) [reply tasmota/tasmota_xnrg_energy/xnrg_12_solaxX1.ino:51]: In general I agree with you. For saving code space, I introduced the compiler option `#define SOLAXX1_READCONFIG `. Disabling this, will saves more than 3k space. So users, who need the space, must not use this unimportant function.

Comment (Jason2866) [reply tasmota/tasmota_xnrg_energy/xnrg_12_solaxX1.ino:51]: Please disable this define by default. Thx.

Comment (SteWers) [reply tasmota/tasmota_xnrg_energy/xnrg_12_solaxX1.ino:51]: OK. Done.

**Pull request merged**